### PR TITLE
All requirements in .txt files

### PR DESCRIPTION
**Pull request by maaktweluit**
Put all requirements in `requirements.txt` files.
- No more manual requirements installed on buildbot
- Removed `-winpackager`
- Added more generic `-build`

Requires buildbot to be updated when this PR hits develop.

### CODE REVIEW

**Review by mfranciszkiewicz — APPROVED** [review]
LGTM

File: requirements.txt [hunk 1]
```diff
@@ -36,6 +36,8 @@ pydispatcher
 Crypto
 scrypt
 packaging
+# GitPython < 2.1.8 has problems with latest git version
+GitPython>=2.1.8
```
Comment (jiivan) [inline requirements.txt]: Is this used anywhere outside buildbot? Maybe `requirements-build.txt` would be better place for it.

Comment (maaktweluit) [reply requirements.txt]: Good catch! it was used for getting the version, will double check

Comment (maaktweluit) [reply requirements.txt]: You are right, not used at all anymore, will remove it.

**Review by jiivan — APPROVED** [review]

**Pull request merged**